scipy/stats/_hypotests.py

```diff
@@ -1861,6 +1872,9 @@ def tukey_hsd(*args):
     sample1, sample2, ... : array_like
         The sample measurements for each group. There must be at least
         two arguments.
+    equal_var: bool, optional
+        If True (default), perform Tukey-HSD test [6]. If False, perform
```

Comment:
Probably should include the information that you do below: if `equal_var=True`, it uses Tukey-Kramer [4] if the sample sizes are unequal.